.github/workflows/run_tests_prod.yml

```diff
@@ -0,0 +1,15 @@
+name: Run Tests - Prod
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 6,18 * * *'
```

Comment:
Sets up a cron job to automatically run the workflow twice a day, at 6:00 AM and 6:00 PM (UTC)